mod-invoice-storage/examples/voucher_collection.sample

```diff
@@ -0,0 +1,39 @@
+{
+  "vouchers": [
+     {
```

Comment:
Spacing seems off here. I think this complete object can be pulled back by 2 spaces.

Reply:
I think this complete object can be pulled back by 1 space, but I also don't think it really matters all that much in these examples, as long as it's readable and consistent within the file.  Looking at the examples across acq-models there isn't much consistency wrt indentation of array elements.